evaluation.md

```diff
@@ -5,6 +5,31 @@
 _Describe each change from your ideal syntax to the syntax you implemented, and
 describe_ why _you made the change._
 
+First, we added a package line and put all the code in an object, because we are
+pretty sure they are mandatory. We got rid of the curly braces around the state
+declarations because we realized we could go without them and decided they were
+not the most beautiful. We added "and" between the direction and the next state
```

Comment:
Did you consider swapping `and` for `then`? In my mental model of Picobot, I interpret Picobot's movement and state change as happening in sequentially - though I suppose in reality they are meant to be simultaneous, I don't think that interpreting the actions as sequential makes the mental model any less useful (and at least for me, it was simpler to think of it that way).